### PR TITLE
Batch ES stats extration

### PR DESCRIPTION
**Pull request by cableman**
https://jira.itkdev.dk/browse/DDBTEAM-413

### CODE REVIEW

File: src/Service/ElasticSearchService.php [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+    private $elasticSearchURL;
+
+    /** @var HttpClientInterface $httpClient */
+    private $httpClient;
+
+    /** @var FilesystemAdapter $cache */
+    private $cache;
+
+    /**
+     * StatisticsExtractionService constructor.
```
Comment (tuj) [inline src/Service/ElasticSearchService.php:36]: copy/paste

File: src/Service/ElasticSearchService.php [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+            'body' => $jsonQuery,
+            'headers' => [
+                'Content-Type: application/json',
+                'Content-Length: '.strlen($jsonQuery),
+            ],
+        ]);
+        $results = json_decode($response->getContent());
+
+        // Save the current scroll id (it might change from request to request).
+        $this->getScrollId($results->_scroll_id);
```
Comment (tuj) [inline src/Service/ElasticSearchService.php]: does not look like a save function

File: .env [hunk 3]
```diff
@@ -34,7 +34,7 @@ CORS_ALLOW_ORIGIN='^https?://localhost(:[0-9]+)?$'
 ###< nelmio/cors-bundle ###
 
 ###> app ###
-ELASTICSEARCH_URL='http://elasticsearch:9200/'
+ELASTICSEARCH_URL='http://host.docker.internal:9200/'
```
Comment (tuj) [inline .env]: Change back

**Review by tuj — CHANGES_REQUESTED** [review]
The exist check for entries seems to be missing. This should also be implemented with a test case.

File: src/Service/ElasticSearchService.php (hunk 1 above)
Comment (cableman) [reply src/Service/ElasticSearchService.php:36]: Fixed :-D

File: src/Service/ElasticSearchService.php (hunk 2 above)
Comment (cableman) [reply src/Service/ElasticSearchService.php]: Added new set and get functions to make it more clear.

File: .env (hunk 3 above)
Comment (cableman) [reply .env]: Fixed

**Review by tuj — APPROVED** [review]

**Pull request merged**